.pipelines/cni/cilium/cilium-overlay-load-test-template.yaml

```diff
@@ -93,7 +93,14 @@ stages:
                   kubectl apply -f test/integration/manifests/cilium/v${DIR}/cilium-agent/files
                   kubectl apply -f test/integration/manifests/cilium/v${DIR}/cilium-operator/files
 
-                  envsubst '${CILIUM_VERSION_TAG},${CILIUM_IMAGE_REGISTRY}' < test/integration/manifests/cilium/v${DIR}/cilium-agent/templates/daemonset.yaml | kubectl apply -f -
+                  if [ ! -z ${{ parameters.dualstackVersion }} ]; then

```

Comment:
I believe the `!` needs to be outside of the `[ ]` to function properly

Reply:
This order also works, here's output from the pipeline: 
```
+ '[' '!' -z 1.14.10 ']'
+ echo 'Use dualstack daemonset for Cilium'
Use dualstack daemonset for Cilium
++ make ipv6-hp-bpf-version
```